### PR TITLE
Created Login Component

### PR DESCRIPTION
**Pull request by mini024**

### CODE REVIEW

File: client/src/components/Login/Login.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                <CardContent>
+                   <form noValidate autoComplete="off">
+                           <TextField
+                             id="login"
+                             label="Username"
+                             margin="none"
+                             style = {textFieldStyle}
+                           />
+                           <br/>
+                           <TextField
```
Comment (gerardogalvez) [inline client/src/components/Login/Login.js]: `type="password"`

**Review by gerardogalvez — DISMISSED** [review]

File: client/src/components/Login/Login.js (hunk 1 above)
Comment (mini024) [reply client/src/components/Login/Login.js]: Nice catch!

**Comment by mini024** [comment]
Front-end React:
- Created Login Component
- Added axios for http requests

Back-end Flask:
- Created Authentication class to handle user login
- Created Create User class

Database:
- Created Stored Procedures to handle login and
creating new user.

Notes:
- Install axios on react with: npm install axios
- Install flask-restful with: pip install flask-restful
- Run the Procedures.sql in mysql, if there is an
error try doing copy paste instead of running file

**Comment by mini024** [comment]
To test you need to install bcrypt by doing: `pip install bcrypt `

**Comment by mini024** [comment]
Getting the following error 

> Error: Invalid Salt

Talking with @gerardogalvez to fix it.

**Comment by mini024** [comment]
> Getting the following error
> 
> > Error: Invalid Salt
> 
> Talking with @gerardogalvez to fix it.

Problem was that passwords weren't hashed in database.

**Pull request merged**